### PR TITLE
New Bot field: version 

### PR DESCRIPTION
**Pull request by kamilsmuga**
Added a new field - version - for Bot model. This will allow better tracking of Bot releases as well as implementing a solid caching strategy

### CODE REVIEW

**Comment by iamigo** [comment]
please don't add reviewers until tests are passing :(

**Comment by kamilsmuga** [comment]
@iamigo tests fixed - they caught a lot of cases. Nice!

File: api/v1/swagger.yaml [hunk 1]
```diff
@@ -2058,6 +2058,11 @@ paths:
           in: formData
           type: file
           format: binary
+        -
+          name: version
```
Comment (iamigo) [inline api/v1/swagger.yaml]: required field?

File: migrations/20180323141450-add-column-bot-version.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up(qi, Sequelize) {
```
Comment (iamigo) [inline migrations/20180323141450-add-column-bot-version.js]: do you need to backfill existing rows in the migration too?

**Pull request merged**